### DISCUSSION TITLE
Welcome to Win11ClockToggler Discussions!

### DISCUSSION THREAD

**Discussion by jmalarcon** [jmalarcon/Win11ClockToggler#6] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with other members of the community. Use this are to:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Tell me how you're using the app

>Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and how you are using the app.

Enjoy!

**Comment by alecthegeek** [comment]
Thank you for clock toggler. I use it when I do desktop recordings so the content is not dated

I am quite technical so I don't mind using the CLI version

**Reply by jmalarcon** [reply]
I'm glad that you are taking good advantage of it.

Please, take into account that the GUI version monitors the changes in the notification area to hide again it if anything changes (i.e. if a new notification icon is shown, Windows will show the whole area automatically). The CLI version does not control this.

Thanks for commenting.

Enjoy.

**Comment by Virtualburn** [comment]
I love it when the Open Source Community are the ones to come to the rescue when MS and all it's resources fail people.  This wasn't an intentional upgrade and it's broken most thing I use for work.  I live in EU but work by UK time so this is really important for me.

Not sure how it even disappeared as it was there just after the upgrade.

I've tried your tool and the first 'Date/Time' is grayed out along with the third option and I can only use the 'Notification Area' toggle.  I'd be happy to supply any local details / logs to help improve your solution if needed.

http://prntscr.com/8uF5Pai20H7o

darren [at] bootloop [dot] com

**Reply by jmalarcon** [reply]
Hi Darren:

Thanks for commenting.

What you show in your screen capture is the expected behavior. 

- The first toggle is to hide the date/time, and it's always checked (and therefore disabled) because this is the main purpose of the tool: to hide the date/time area. 
- The second toggle is to hide the notifications area. Is checked by default because I think is the best way to go for recording your screen, which is the main usage I had in mind for the tool, but it'll keep your last selection (enabled/disabled) the next time you run the app. 
- The third one is only enabled if you have at least a secondary screen with the taskbar enabled in it. If that's not the case, it is disabled because there's nothing to hide.

HTH

**Reply by Virtualburn** [reply]
Thanks.

Well I'm not sure what it did but it worked.  After a week now of having no time and trying everything to restore it ... it's back.  Within 5 minutes of playing around with your tool, there seemed to be a delay but it's fixed.   That's great!

After using your tool and going to the time / date settings it just seemed to appear.

**Comment by DMSRoss** [comment]
OK. I am a newby. The app is great idea but it doesn't hide the clock.  Why?

**Reply by jmalarcon** [reply]
Hi: it hides the clock in **Windows 11** and Windows 10 too. It's its main purpose. Why do you say it doesn't? Have you clicked the `Toggle visibility` button?

If something is not working on your end, please, share a short video or animation and I'll try to help you with what's happening. Thanks.

**Comment by DMSRoss** [comment]
See attached.  Yes, I toggled.  Still there.

 Dr. M. Steven Ross

www.msrministry.com

**Reply by jmalarcon** [reply]
No attachment in your email (neither in the resulting message here). Anyway, I need much more information: your exact Windows version, language, version of Win11ClockToggler... And the screen capture with the clock will be useful too (please, use directly GitHub and not the email).

The software works, so there's something related to your computer. Try it in a different computer or virtual machine and see if it works there.
